app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventInitial/EventInitialPresenter.java

```diff
@@ -238,6 +238,7 @@ private void getProgramStages(String programUid, String programStageUid) {
 
     @Override
     public void onBackClick() {
+        preferences.setValue(Preference.EVENT_COORDINATE_CHANGED, false);
```

Comment:
do we need to clear this values?

Reply:
Yes. Clearing or setting it to false will do the same.